subsys/dfu/boot/mcuboot.c

```diff
@@ -252,6 +253,14 @@ static int boot_copy_done_read(u32_t bank_offs)
 	return boot_flag_read(BOOT_FLAG_COPY_DONE, bank_offs);
 }
 
+static int boot_version_read(u32_t bank_offs, struct image_version *out_ver)
```

Comment:
substitute is already implemented on master see https://github.com/zephyrproject-rtos/zephyr/pull/5945/files
similar remark to f.  boot_magic_code_check, f. boot_magic_state_read, f. boot_current_image_version